### PR TITLE
feat:login-signup pages added

### PR DESCRIPTION
**Pull request by jatinmore**
Login and Signup Pages added

### CODE REVIEW

**Comment by ArpitPatni** [comment]
ss3

**Review by arulk8 — COMMENTED** [review]
Looks good. the alignment of name and Email in signup form is little different.

**Comment by ArpitPatni** [comment]
Bro password and confirm password line is not aligned to the right with name and email.
Signup button can be aligned to centre as well.

**Review by harshitpaliwal95 — APPROVED** [review]
looks good

File: pages/login-signup/signup.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    </nav>
+
+    <div class="container">
+        <div class="items">
+            <div class="card card-login">
+                <div class=" card-body description">
+                    <label for="">Name<input type="text" class="input-field outline" required></label>
+                    <label for="">Email<input type="email" class="input-field outline"required></label>
+                    <label for="">Password<input type="password" class="input-field outline"required></label>
+                    <label for="">Confirm Password<input type="password" class="input-field outline"required></label>
```
Comment (ArpitPatni) [inline pages/login-signup/signup.html]: Please name password and confirm password right aligned as it not aligning with name and email

**Pull request merged**